xunit.runner.wpf/FilteredCollectionView.cs

```diff
@@ -283,7 +283,13 @@ void IList.RemoveAt(int index)
 
         object IList.this[int index]
         {
-            get { return this[index]; }
+            get
+            {
+                // Can't figure out how to not get a warning here.
+#pragma warning disable CS8603
+                return this[index];
+#pragma warning restore
```

Comment:
Could add a `?? throw` here to remove the exception. That will function. Don't think that can be a shipping solution though.